mteb/evaluation/evaluators/Image/Any2AnyRetrievalEvaluator.py

```diff
@@ -121,11 +125,14 @@ def search(
         if q_modality == "text":
             query_texts = queries["text"]
             query_embeddings = self.model.get_text_embeddings(
```

Comment:
```suggestion
            query_embeddings = self.model.encode
```

Reply:
(general)

Reply:
Hi, I get your point, `get_text_embeddings` and `encode` are functionally the same.
But I would prefer using `get_text_embeddings` since most multimodal embedding models don't implement `encode`.

Reply:
Feel free to leave it as is, but I believe the hope is to move them to the encode() interface (see #1551)